nemo_curator/stages/deduplication/semantic/workflow.py

```diff
@@ -380,7 +380,7 @@ def run(self, pairwise_executor: BaseExecutor | None = None) -> dict[str, Any]:
             if total_duplicates > 0:
                 logger.success(f"Total documents identified as duplicates: {total_duplicates}")
                 logger.info(f"Similarity threshold used: {1.0 - self.eps:.3f} (eps={self.eps})")
-            else:
+            elif self.eps is not None:
```

Comment:
This was a bug since when epis is None then the logger fails since it does `{1.0 - self.eps}`